src/audio_call.h

```diff
@@ -44,14 +44,28 @@ typedef enum _VideoError {
 
 #endif /* VIDEO */
 
+/* Status transitions:
+ * None -> Pending (call invitation made or received);
+ * Pending -> None (invitation rejected or failed);
+ * Pending -> Active (call starts);
+ * Active -> None (call ends).
+ */
+typedef enum _CallStatus {
```

Comment:
This name is reserved for the C implementation (ISO C 7.1.3). I'd just use `CallStatus`.